test/01_TestOracle.test.js

```diff
@@ -48,50 +47,30 @@ contract('Oracle pricing', (accounts) => {
   const usdcEthCumPrice1_1 = new BN('31378725947216452626380862836246')
   const usdcEthTimestamp_1 = new BN('1606781003')
   const usdcEthTokensInReverseOrder = true
-  const usdcEthScaleFactor = (new BN(10)).pow(ethDecimals.add(new BN(18)).sub(usdcDecimals))
-
-  const daiEthCumPrice0_0 = new BN('291033362911607134656453476145906896216') // From the DAI/ETH pair
-  const daiEthCumPrice1_0 = new BN('30339833685805974401597062880404438583745289')
-  const daiEthTimestamp_0 = new BN('1606780728')
-  const daiEthCumPrice0_1 = new BN('291036072023637413832938851532265880018')
-  const daiEthCumPrice1_1 = new BN('30340852730044753766501469633003499944051151')
-  const daiEthTimestamp_1 = new BN('1606781048')
-  const daiEthTokensInReverseOrder = true
-  const daiEthScaleFactor = (new BN(10)).pow(ethDecimals.add(new BN(18)).sub(daiDecimals))
```

Comment:
Again, I'd probably keep these in case we need to swap in one of the other pairs sometime, but I could be convinced!  I wonder what @albertocuestacanada thinks.

Reply:
If we need this in the future we could go back in git history or prehaps move it into a constants file